develop-docs/self-hosted/experimental/monitoring.mdx

```diff
@@ -0,0 +1,182 @@
+# SENTRY_METRICS_PREFIX = "sentry."  # Adjust this to your needs, default is "sentry."
+```
+
+### Snuba
+
+You can configure Snuba to send metrics to Statsd server by configuring your `docker-compose.yml` file:
+
+```yaml
+# The rest of your Docker Compose file
+
+x-snuba-defaults: &snuba_defaults
+  <<: *restart_policy
+  depends_on:
+    clickhouse:
+      <<: *depends_on-healthy
+    kafka:
+      <<: *depends_on-healthy
+    redis:
+      <<: *depends_on-healthy
+  image: "$SNUBA_IMAGE"
+  environment:
+    SNUBA_SETTINGS: self_hosted
+    CLICKHOUSE_HOST: clickhouse
+    DEFAULT_BROKERS: "kafka:9092"
+    REDIS_HOST: redis
+    UWSGI_MAX_REQUESTS: "10000"
+    UWSGI_DISABLE_LOGGING: "true"
+    # Leaving the value empty to just pass whatever is set
+    # on the host system (or in the .env file)
+    SENTRY_EVENT_RETENTION_DAYS:
+    DOGSTATSD_HOST: "statsd.yourcompany.com"
+    DOGSTATSD_PORT: 8125
+
+# The rest of your Docker Compose file
+```
+
+### Relay
+
+You can configure Relay to send metrics to Statsd server by configuring your `relay/config.yml` file:
+
+```yaml
+# The rest of your config.yml file
+metrics:
+  statsd: "statsd.yourcompany.com:8125"
+  # prefix: "sentry.relay" # Adjust this to your needs, default is "sentry.relay"
+  # sample_rate: 1.0 # Adjust this to your needs, default is 1.0
+  # `periodic_secs` is the interval for periodic metrics emitted from Relay.
+  # Setting it to `0` seconds disables the periodic metrics.
+  # periodic_secs: 5
+```
+
+### Symbolicator
+
+You can configure Symbolicator to send metrics to Statsd server by configuring your `symbolicator/config.yml` file:
+
+```yaml
+# The rest of your config.yml file
+metrics:
+  statsd: "statsd.yourcompany.com:8125"
+  prefix: "sentry.symbolicator" # Adjust this to your needs, default is "symbolicator"
+```
+
+## Sentry dependencies
+
+Sentry and its' surrounding services have some dependencies like Redis, ClickHouse, Kafka, etc, that you can monitor with your monitoring system. The easiest way to do so is through Prometheus exporters. Although there are many exporters available, for reference purposes, you can use this setup below.
+
+Modify your `docker-compose.yml` file to add the following:
+
+```yaml
+services:
+  # The rest of your Docker Compose file
+  prometheus-postgresql-exporter:
+    <<: *restart_policy
+    image: quay.io/prometheuscommunity/postgres-exporter
+    ports:
+      - "127.0.0.1:9187:9187"
+    environment:
+      DATA_SOURCE_URI: "postgres:5432/postgres?sslmode=disable"
+      DATA_SOURCE_USER: "postgres"
+  prometheus-redis-exporter:
+    <<: *restart_policy
+    image: oliver006/redis_exporter
+    ports:
+      - "127.0.0.1:9121:9121"
+    environment:
+      REDIS_ADDR: "redis:6379"
+  prometheus-kafka-exporter:
+    <<: *restart_policy
+    image: danielqsj/kafka-exporter
+    ports:
+      - "127.0.0.1:9308:9308"
+    command: ["--kafka.server=kafka:9092"]
```

Comment:
I was a bit hesitant to use this, since most people recommend this: https://github.com/prometheus/jmx_exporter

Personally I don't use both, since I use Redpanda, and they already got built-in Prometheus metrics https://docs.redpanda.com/current/manage/monitoring/